src/eq_schema/Answer.test.js

```diff
@@ -37,6 +37,25 @@ describe("Answer", () => {
     expect(answer.currency).toBe("GBP");
   });
 
+  it("should add a min value validation rule", () => {
+    const answer = new Answer(
+      createAnswerJSON({
+        validation: {
+          minValue: {
+            id: "1",
+            inclusive: false,
+            enabled: true,
+            custom: 5
+          }
+        }
+      })
+    );
+    expect(answer.min_value).toMatchObject({
+      value: 5,
+      exclusive: true
+    });
+  });
+
```

Comment:
Worth adding a test for when inclusive is set to true and another for when the validation rule is disabled? Otherwise looks good to me.